bodyfetcher.py

```diff
@@ -124,9 +134,16 @@ def make_api_call_for_site(self, site):
                 pagesize = "50"
             else:
                 pagesize = "25"
-            url = "http://api.stackexchange.com/2.2/questions?site=stackoverflow&filter=!)E0g*ODaEZ(SgULQhYvCYbu09*ss(bKFdnTrGmGUxnqPptuHP&key=IAkbitmze4B8KpacUfLqkw((&pagesize=" + pagesize + min_query
+            pagesize_modifier = "&pagesize={0}{1}".format(pagesize, min_query)
```

Comment:
I think this would have been better if

1. The arguments to `format` where named, so we weren't dealing with `{0}` and `{1}`
2. It was somehow made more clear what `min_query` is and why it is being put at the very end of this

As it currently stands, I think most people could figure out what `&pagesize={0}` does. The trailing `{1}` is strange though.